flocker/apiclient/_client.py

```diff
@@ -0,0 +1,90 @@
+# Copyright ClusterHQ Inc.  See LICENSE file for details.
+
+"""
+Client for the Flocker REST API.
+"""
+
+from uuid import UUID
+
+from zope.interface import Interface
+
+from pyrsistent import PClass, field, pmap_field
+
+
+class Dataset(PClass):
```

Comment:
:question: And can't we re-use the existing types from `flocker.control._model` here? Or are you anticipating this package being installed without a `Flocker` dependency?